### ISSUE TITLE
Stop button on the "All file" page

### ISSUE THREAD

**Issue by jameslao** [nextcloud/server#16137]
Using Nextcloud snap 17.0.0 alpha Build:2019-06-26T22:01:48+00:00 

If I go to pages like Recent, Favorites, Shares, etc, and come back to "All Files" page, there's a stop button shown, like shown below.

![image](https://user-images.githubusercontent.com/5279743/60301801-1e04a900-98e7-11e9-9eb7-a5a8f62e5e0b.png)

Refreshing the page removes it. 

This button usually has `style="display:none"`, but that style is missing if I switch pages as mentioned above.

```
<div id="uploadprogresswrapper">
<div id="uploadprogressbar" data-original-title="" title="">
	<em class="label outer">
	<span class="desktop"></span>
	<span class="mobile"></span></em>
</div>
<button class="stop icon-close" style=""> 
	<span class="hidden-visually"></span>
</button>
</div>
```

**Comment by jancborchardt** [comment]
@jameslao @kesselb want to submit a pull request to fix this? :) It’s part of the polishing list for Nextcloud 17: https://github.com/nextcloud/server/issues/16076

**Comment by kesselb** [comment]
Don't know how to fix this :see_no_evil: Just added the right labels and referenced it in #16076.

**Comment by jameslao** [comment]
I'll take a shot at it.

**Comment by MorrisJobke** [comment]
Fixed by #16151
